client/post-editor/editor-drawer/featured-image.jsx

```diff
@@ -64,6 +64,7 @@ class EditorDrawerFeaturedImage extends Component {
 						site={ site }
 						post={ post }
 					/>
+					{/* todo maybe change this to use dropzone as well? why would the dropzone be attached to the EditorDrawerWell instead of the FeaturedImage? */}
```

Comment:
I think I added it here because there was a styling problem, i.e. the DropZone would only stretch over the middle area, without covering the whole "block". 

It can definitely be improved and it would be awesome if we can get this in as a separate, refactor PR!